docs/elements.md

```diff
@@ -262,15 +262,15 @@ Attribute | Type | Default | Description
 --- | --- | --- | ---
 `answers-name` | string | — | Variable name to store data in.
 `weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
-`correct-answer` | float | special | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.
+`correct-answers` | float | special | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.
```

Comment:
This is the argument expected by `pl-number-input`, it was correct originally. The problem was in the Python version, not the HTML argument.

Reply:
I think I am not understanding something crucial... 

For numeric input, `correct-answer` produces an error in the problem.

<img width="813" alt="Screen Shot 2021-05-08 at 2 19 31 PM" src="https://user-images.githubusercontent.com/2507459/117553810-74c18a80-b008-11eb-85a2-478705fb5d42.png">